grub-core/loader/multiboot_mbi2.c

```diff
@@ -301,10 +307,10 @@ grub_multiboot2_load (grub_file_t file, const char *filename)
 	      return grub_error (GRUB_ERR_BAD_OS, "invalid min/max address and/or load size");
 	    }
 
-	  err = grub_relocator_alloc_chunk_align_safe (grub_multiboot2_relocator, &ch,
```

Comment:
Let's keep the safe function

Reply:
@miczyg1 changed to safe, and code_size substraction removed https://github.com/TrenchBoot/grub/pull/8/commits/c3f98c34e3111453bfdd7a2f57adb03bcd848453#diff-285279252527f5bc911013189ac09a0880271f60dc8f28f28fe68c2f5d517c74R310